test/system/toolbar_test.rb

```diff
@@ -165,4 +200,13 @@ class ToolbarTest < ApplicationSystemTestCase
     click_on "Redo"
     assert_equal_html "<p>Hello World</p>", find_editor.value
   end
+
+  private
+    def apply_highlight_option(attribute)
+      click_on "Color highlight"
+
+      within "lexxy-color-dialog dialog[open] [data-button-group='#{attribute}']" do
+        first(".lexxy-color-button").click
```

Comment:
you could pass a `button_index` as the second param and here do:

```ruby
all(".lexxy-color-button")[button_index - 1].click # So 1 is the first button.
```

So that it is clear which button you are invoking when you invoke the helper